run_environment.sh

```diff
@@ -572,7 +572,7 @@ AIRFLOW_BREEZE_TEST_SUITE=${ASCII_USER:0:6}${NUMERIC_PYTHON_VERSION}
 RANDOM_FILE=${MY_DIR}/.random
 
 if [[ ! -f ${RANDOM_FILE} ]]; then
-    head /dev/urandom && true | env LC_CTYPE=C tr -dc 'a-z0-9' | fold -w 7 | head -n 1 > ${RANDOM_FILE}
```

Comment:
Why do we need that @mik-laj - I thought -c 7 does the job.

Reply:
Code readability only. instead of two commands we can use only one. Is the world beautiful now?

Reply:
It's been already even more simplified in master (using date | md5sum which is much better as it does not require randomness of /dev/urandom)